### ISSUE TITLE
"Encrypt" and "Trust Server Certificate" tooltip is not accessible using keyboard: A11y_MySQL extension for Azure Data Studio_Connect to a MySQL Server_Keyboard

### ISSUE THREAD

**Issue by Swati1700** [microsoft/azuredatastudio#25560]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”
**Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification.**

### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM_MySQL extension for Azure Data Studio_Win32_Mar2024; #MySQL extension for Azure Data Studio; #Win32; #Benchmark; #Win11; #WCAG2.1.1; #Keyboard; #FTP;

### Environment and OS details:
Azure Data Studio Version: 1.48.0 (user setup)
Windows 11 pro Version 23H2
OS build 22631.3296

### Repro Steps:

1. Launch Azure Data Studio.
2. Tab to the new connection button and press enter.
3. Tab to "Encrypt" and "Trust Server Certificate" and observe tooltip is not accessible using keyboard.

### Actual Result:
"Encrypt" and "Trust Server Certificate" tooltip is not accessible using keyboard.

### Expected Result:
"Encrypt" and "Trust Server Certificate" tooltip should be accessible using keyboard.

### User Impact:
Keyboard users and screen reader users will not get tooltip information if it is not accessible.

### Attachment:
[Keyboard.zip](https://github.com/microsoft/azuredatastudio-mysql/files/14722406/Keyboard.zip)

**Comment by rishky-msft** [comment]
@Swati1700,

This issue is not in MySQL extension. The video that you have shared is for SQL server connection dialog.
I have transferred the issue to azuredatastudio repo. Please remove this from the MySQL extension accessibility items.

**Comment by Swati1700** [comment]
@rishky-msft We cannot directly close the issue from accessibility items and as this is external to azuredatastudio we can log a external bug for it.

**Comment by rishky-msft** [comment]
@Swati1700,

This bug is not external.

Even this is not a bug in MySQL extension.
The properties mentioned "Encrypt" and "Trust Server Certificate" are not available for MySQL connection type.

You can verify and close this bug from MySQL extension.

**Comment by Swati1700** [comment]
As per the above comment issue does not repro in MYSQL extension, hence closing the bug.
